clean/lake_charles_pd_cprr.py

```diff
@@ -0,0 +1,149 @@
+import sys
+sys.path.append('../')
+import pandas as pd
+from lib.path import data_file_path, ensure_data_dir
+from lib.clean import float_to_int_str
+from lib.columns import clean_column_names
+from lib.rows import duplicate_row
+from lib.uid import gen_uid
+import re
+
+
+def clean_charges(df):
+    df.loc[:, 'charges'] = df.nature_of_complaint.str.lower().str.strip()\
+        .str.replace(r'\. ?', ' ', regex=True)\
+        .str.replace(',', '', regex=False)\
+        .str.replace(r'(\d+) ', '', regex=True)\
+        .str.replace('unsat perform', 'unsatisfactory performance', regex=False)\
+        .str.replace('neg of duty', 'neglect of duty', regex=False)\
+        .str.replace(r'cond unbecom(ing)?', 'conduct unbecoming', regex=True)\
+        .str.replace('unauth force', 'unauthorized force', regex=False)\
+        .str.replace('use of dept equip', 'use of department equipment', regex=False)\
+        .str.replace('unknown', '', regex=False)
+    return df.drop(columns='nature_of_complaint')
+
+
+def split_rows_with_multiple_charges(df):
+    i = 0
+    for idx in df[df.charges.str.contains(r'/')].index:
+        s = df.loc[idx + i, 'charges']
+        parts = re.split(r'\s*(?:/)\s*', s)
+        df = duplicate_row(df, idx + i, len(parts))
+        for j, name in enumerate(parts):
+            df.loc[idx + i + j, 'charges'] = name
+        i += len(parts) - 1
+    return df
+
+
+def clean_action(df):
+    df.loc[:, 'action'] = df.action_taken.str.lower().str.strip()\
+        .str.replace(r'no action ?(tak[eo]n)?', '', regex=True)
+    return df.drop(columns=('action_taken'))
+
+
+def consolidate_action_and_disposition(df):
+    df.loc[:, 'action'] = df.action.str.cat(df.disposition, sep='|')\
+        .str.replace(r'(not)? ?sustained ?|exonerated ?|unfounded ?|invalid complaint ?', '', regex=True)\
```

Comment:
slightly shorter pattern: `r'((not)? ?sustained|exonerated|unfounded|invalid complaint) ?'`